entity-service/src/main/kotlin/com/egm/stellio/entity/service/EntityService.kt

```diff
@@ -276,9 +286,15 @@ class EntityService(
                 (it["property"] as Property).id
             }
             .values
+            .map { buildInstanceFragment(it, entity.contexts) }
+            .groupBy { it.first }

```

Comment:
To think of it again, I think it would be fine to replace it by variables here to improve readibility... What do you think ?

Reply:
Yes